src/js/claims-status/reducers/appeals.js

```diff
@@ -4,27 +4,65 @@ import {
   SET_APPEALS,
   SET_APPEALS_UNAVAILABLE,
   FETCH_APPEALS_PENDING,
-  FETCH_APPEALS_SUCCESS
+  FETCH_APPEALS_SUCCESS,
 } from '../actions/index.jsx';
 
+import {
+  USER_FORBIDDEN,
+  RECORD_NOT_FOUND,
+  VALIDATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  FETCH_APPEALS_ERROR,
+  AVAILABLE
+} from '../utils/appeals-v2-helpers';
+
 const initialState = {
   available: true,
+  availabilityError: null,
 };
 
 export default function appealsReducer(state = initialState, action) {
   switch (action.type) {
     case FETCH_APPEALS_PENDING:
       return _.set('appealsLoading', true, state);
-    case FETCH_APPEALS_SUCCESS:
+    case FETCH_APPEALS_SUCCESS: // Appeals v1 and v2
+      // appeals are added to Redux store via FETCH_APPEALS_SUCCESS
+      // reducer in claims-list.js
       return _.merge(state, {
         appealsLoading: false,
         available: true,
+        v2Availability: AVAILABLE // New and improved! More bits of info!
       });
-    case SET_APPEALS:
+    case SET_APPEALS: // Appeals v1
       return _.set('available', true, state);
-    case SET_APPEALS_UNAVAILABLE:
-      // Maybe should set appealsLoading to false here too
+    case SET_APPEALS_UNAVAILABLE: // Appeals v1
       return _.set('available', false, state);
+    // Following are reducers for Appeals v2 error states  
+    case USER_FORBIDDEN:
+      return _.merge(state, {
+        appealsLoading: false,
+        v2Availability: USER_FORBIDDEN,
```

Comment:
If we keep the v1 and v2 reducer functionality in the same spot, does it make sense to set `available` to `false` here for these error states?